From 020fc5befa8baea6f89241ef6f5d11f3e70d7a21 Mon Sep 17 00:00:00 2001
From: Shalini Salomi Bodapati <shalini.salomi.bodapati@intel.com>
Date: Mon, 11 Feb 2019 16:01:19 +0530
Subject: [PATCH] Add mixins update for adding new cpu arch for cel_kbl
 Tracked-On: OAM-76094 Signed-off-by: Shalini Salomi Bodapati
 <shalini.salomi.bodapati@intel.com>

---
 cel_kbl/mixins.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cel_kbl/mixins.spec b/cel_kbl/mixins.spec
index 38d2453..ebadd1a 100644
--- a/cel_kbl/mixins.spec
+++ b/cel_kbl/mixins.spec
@@ -24,7 +24,7 @@ bluetooth: btusb (ivi=true)
 boot-arch: project-celadon(bootloader_policy=0x0,bootloader_len=60,magic_key_timeout=80,assume_bios_secure_boot=true,tos_partition=true,rpmb_simulate=true,disk_encryption=false,file_encryption=true,ignore_not_applicable_reset=true,self_usb_device_mode_protocol=true)
 audio: project-celadon
 wlan: iwlwifi
-cpu-arch: skl
+cpu-arch: kbl
 cpuset: autocores
 rfkill: true(force_disable=)
 dexpreopt: enabled
-- 
1.9.1

